frontend/src/app/footer/footer.component.html

```diff
@@ -21,7 +21,7 @@ <h3 class="nav-title">Strefa organizacji i instytucji</h3>
             <a class="nav-link" href="{{ djangoRoot }}/register">Załóż konto</a>
             <a class="nav-link" href="{{ djangoRoot }}/offers/create">Dodaj nową ofertę</a>
             <a class="nav-link" href="{{ djangoRoot }}/organizations">Lista organizacji i instytucji</a>
-            <a class="nav-link" href="{{ djangoRoot }}/pages/faq-organizations">FAQ dla organizacji i instytucji</a>
+            <a class="nav-link" href="/faq-organization">FAQ dla organizacji i instytucji</a>
```

Comment:
It should use [RouterLink](https://angular.io/api/router/RouterLink) as @TrebuhD nad @JacSta done in theirs pull requests (see #672 and #675). There's an issue with testing this (that we need to resolve) but `RouterLink` is definitely way to go with it.